debian/postinst

```diff
@@ -53,6 +53,10 @@ autofs_handle_srv() {
 	return $?
 }
 
+decompress_lz4_kernel() {
+	decompress-kernel "/boot/vmlinuz-$(uname -r)"
```

Comment:
I think we should try to decompress all existing vmlinuz files to allow this to work if a more recent, LZ4 compressed kernel is installed but not already used without having to reinstall the kernel.

Reply:
I added this feature, it will now decompress all kernels upon package installation.